### DISCUSSION TITLE
100% Markdown Expedition

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#227] [Announcements]
Hey All,

We have officially launched the project to get to 100% Markdown across all active languages. You can read about the why and how to contribute in [this Hacks blog post](https://hacks.mozilla.org/2022/09/the-100-percent-markdown-expedition/).

**Comment by schalkneethling** [comment]
Hey All,

Latest update on the project. You all are doing fantastic work!!

- es - 1975 > 1723
- ko - 332 > 329
- pt-br - 2071 > 1684
- ru - 2389 > 2389
- fr - 100% Markdown
- ja - 100% Markdown
- zh-cn - 100% Markdown
- zh-tw - 100% Markdown

There are currently [77 open pull requests](https://github.com/mdn/translated-content/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Amarkdown-conversion+) converting content to Markdown

This is the command I use to get the above stats: `find files/zh-tw -type f -name "*.html" | wc -l`

**Reply by SphinxKnight** [reply]
cc @mdn/yari-content-es , @mdn/yari-content-ko, @mdn/yari-content-pt-br , @mdn/yari-content-ru (for notification)

I've opened https://github.com/mdn/translated-content/issues/7580 for `ru` and https://github.com/mdn/translated-content/issues/7950 for `pt-BR`

**Reply by SphinxKnight** [reply]
https://github.com/mdn/translated-content/issues/8635 for `es`

**Reply by SphinxKnight** [reply]
counters on https://github.com/mdn/translated-content/issues/7486 updated as well

**Reply by Graywolf9** [reply]
Thank you so much!

**Comment by schalkneethling** [comment]
Hey All,

This is the status of the project on 27 September 2022. You all are doing fantastic work!!

es - 1723 > 1410 (-313)
ko - 329
pt-br - 1684 > 476 (-1208)
ru - 2389 > 2384 (-4)
fr - 100% Markdown
ja - 100% Markdown
zh-cn - 100% Markdown
zh-tw - 100% Markdown

There are currently [48 open pull requets](https://github.com/mdn/translated-content/pulls?q=is:open+is:pr+label:markdown-conversion) content to Markdown. You all rock 👍

**Comment by schalkneethling** [comment]
Hey All,

This is the status of the project on 4 October 2022. You all are doing fantastic work!!

es - 1410 > 992 (-418)
ko - 329 > 0 (-329) 100% Markdown
pt-br - 476 > 411 (-65)
ru - 2384 > 2384
fr - 100% Markdown
ja - 100% Markdown
zh-cn - 100% Markdown
zh-tw - 100% Markdown

There are currently [33 open pull requets](https://github.com/mdn/translated-content/pulls?q=is:open+is:pr+label:markdown-conversion) content to Markdown. Thank you!! We are so close to 100% Markdown. This is amazing.

**Reply by SphinxKnight** [reply]
only `conflicting` and `orphaned` remain in HTML for `pt-BR` as https://github.com/mdn/translated-content/issues/7950 is now complete :)

**Comment by schalkneethling** [comment]
Hey All,

This is the status of the project on 12 October 2022. You all are doing fantastic work!!

es - 992 > 892 (-100)
pt-br - 411 > 74 (-337) - As mentioned by @SphinxKnight, the remainder is in orphaned and conflicting
ru - 2384 > 2355 (-29)
ko - 100% Markdown
fr - 100% Markdown
ja - 100% Markdown
zh-cn - 100% Markdown
zh-tw - 100% Markdown

There are currently [16 open pull requets](https://github.com/mdn/translated-content/pulls?q=is:open+is:pr+label:markdown-conversion) content to Markdown. Thank you!! We are so close to 100% Markdown. This is amazing.

**Reply by schalkneethling** [reply]
Hey @mdn/yari-content-ru If you all need help with pull request reviews or conversion, please let us know. Thanks!

**Reply by sashasushko** [reply]
It will be great 🥺

I started reviews from the oldest one. You could take the newest one. Then we won't interfere to each other.

**Reply by schalkneethling** [reply]
> It will be great 🥺
> 
> I started reviews from the oldest one. You could take the newest one. Then we won't interfere to each other.

Alrighty, let's get these pull requests merged :) - Thanks, @sashasushko.

**Comment by Graywolf9** [comment]
I think `es` is done

**Reply by SphinxKnight** [reply]
Well done :) Thanks a ton @Graywolf9 🙏👌🎉

**Reply by Graywolf9** [reply]
Thank you so much for all your help!

**Comment by SphinxKnight** [comment]
@schalkneethling `ru` is now done
Thanks a lot to @sashasushko :bow: 

Only `orphaned`/`conflicting` remain for `pt-br` and `ru` when running `find files -type f -name "*.html"` 

Please let me know if you wish to convert those as well (since they need to be dealt with anyway, review shouldn't be a too long task here)

**Reply by schalkneethling** [reply]
Oh my goodness, this is fantastic!

**Comment by schalkneethling** [comment]
Hey All,

This is the status of the project on 30 October 2022.

es - 100% Markdown 🎉 (-892)
pt-br - 74 - As mentioned by @SphinxKnight, the remainder is in orphaned and conflicting
ru - 2355 > 92 (-2263) - As mentioned by @SphinxKnight, the remainder is in orphaned and conflicting
ko - 100% Markdown
fr - 100% Markdown
ja - 100% Markdown
zh-cn - 100% Markdown
zh-tw - 100% Markdown

There are currently [0 open pull requests](https://github.com/mdn/translated-content/pulls?q=is:open+is:pr+label:markdown-conversion) converting content to Markdown. Thank you!!

I essence we are at 100% Markdown!! What a way to end the first month of Q4 🎃 - Thank you to each and every person who played a part in getting us here.

**Comment by Rumyra** [comment]
Wohoo! Amazing everyone 🎉

**Comment by SphinxKnight** [comment]
@schalkneethling should this one be marked as answered or is it better to wait for an orphaned/conflicting cleanup?

**Reply by yin1999** [reply]
append: [two PRs](https://github.com/mdn/translated-content/pulls?q=is%3Aopen+is%3Apr+author%3Ayin1999+convert+to+markdown) left for 100% markdown convertion

**Reply by sashasushko** [reply]
I can deal with it. Only please, help me understand, what I should to do with conflicts? Migrate html>md manually?

**Reply by yin1999** [reply]
See convertion guide here: https://github.com/mdn/markdown/blob/main/conversion-guide.md. For unhandled elements, we can manually resolve them as convertable elements (I've done this before for zh-TW and zh-CN in https://github.com/mdn/translated-content/issues/6173)
